src/wp-includes/rest-api/endpoints/class-wp-rest-themes-controller.php

```diff
@@ -50,11 +50,11 @@ public function register_routes() {
 
 		register_rest_route(
 			$this->namespace,
-			'/' . $this->rest_base . '/(?P<name>[\w-]+)',
+			'/' . $this->rest_base . '/(?P<stylesheet>[\w-]+)',
 			array(
 				'args'   => array(
-					'name' => array(
-						'description' => __( 'Slug name of the theme.' ),
+					'stylesheet' => array(
```

Comment:
Why change this name?

Reply:
@spacedmonkey as per https://github.com/WordPress/wordpress-develop/pull/264/files#r449824792